src/bluesky/callbacks/tiled_writer.py

```diff
@@ -0,0 +1,381 @@
+from abc import abstractmethod
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from area_detector_handlers import HandlerBase
```

Comment:
We don't need this. The methods its provides (`__enter__`, `__exit__`, `close`) are not applicable to this use case.

Reply:
Removed